server/__init__.py

```diff
@@ -3,7 +3,7 @@
 from server.common.utils.utils import import_plugins
 
 __version__ = "1.0.1"
-display_version = "cellxgene v" + __version__
+display_version = "CellxGene v" + __version__
```

Comment:
```suggestion
display_version = "cellxgene v" + __version__
```
We can leave code references as-is for now.